__tests__/app.test.js

```diff
@@ -488,6 +487,18 @@ describe("get article by id returns comment count", () => {
   });
 });
 
+describe("get api", () => {
+  it("200: resolves with contents of endpoints.json", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body: { endpoints } }) => {
+        const keys = Object.keys(endpoints);
+        expect(keys.includes("GET /api")).toBe(true);
```

Comment:
I would expect to see a few more available endpoints here. Currently the endpoint is testing for the existence of itself in endpoints.json which is fine but adding additional endpoints would make a stronger test. Well done for not making it too brittle though 👍